package.json

```diff
@@ -11,6 +11,8 @@
     "start": "gatsby develop",
     "start:refresh": "ENABLE_GATSBY_REFRESH_ENDPOINT=true yarn start",
     "refresh": "curl -X POST http://localhost:8000/__refresh",
+    "import:contentful:demo": "contentful space import --config ./contentful/import-demo-config.json",
```

Comment:
Did you try including contentful-cli as a dev dependency to avoid having to install the CLI globally?